modules/hostap/src/supp_api.c

```diff
@@ -2486,6 +2585,12 @@ int supplicant_ap_enable(const struct device *dev,
 		return -1;
 	}
 
+	ret = set_ap_bandwidth(dev, params->bandwidth);
+	if (ret) {
+		wpa_printf(MSG_ERROR, "Failed to set ap bandwidth");
```

Comment:
If other platforms don't support offload `ap_config_params`, then when it starts AP, it will fail and return here?

Reply:
Yes, will update.